locustfile.py

```diff
@@ -18,273 +18,400 @@
 # Avoid "Too many open files" error
 resource.setrlimit(resource.RLIMIT_NOFILE, (9999, 999999))
 
-API_KEY = os.environ['FEC_API_KEY']
+API_KEY = os.environ["FEC_API_KEY"]
```

Comment:
@qqss88 is there any specific reason you changed the string representation from single to double quotes?

Reply:
it is black code formatting preference. I'll disable it in the future. Thx.